Quartus/fixPerm.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# This script fixes permissions if they are ever reset to 000.
+# Useful when using WSL for git and Quartus on Windows.
+
+me=`basename "$0"`
+
+find . -type d -exec chmod 755 {} \;
+find . -type f ! -name "$me" ! -name "compileflash.sh" -exec chmod 644 {} \;
```

Comment:
* replace the `compileflash.sh` with `*.sh`, presumably we want all sh files to be executable
* speaking of which, do we want to add a `find . -type f -name "*.sh" -exec chmod 755 {} +`
* nit: replace the `\;` with `+` since it results in fewer exec invocations which'll make it a lot faster